controllers/cloud.redhat.com/providers/deployment/provider.go

```diff
@@ -17,5 +17,5 @@ func GetDeployment(c *providers.Provider) (providers.ClowderProvider, error) {
 }
 
 func init() {
-	providers.ProvidersRegistration.Register(GetDeployment, 0, ProvName)
+	providers.ProvidersRegistration.Register(GetDeployment, 1, ProvName)
```

Comment:
Changing provider ordering is dangerous!

Reply:
I actually have an idea for a new way of ordering - but it's a little more complex and going to take me some time to figure out.